### PR TITLE
Chore - Set up sass

### PR DESCRIPTION
**Pull request by ginabeki**
#### What does this PR do?
- Sets up SASS
- Adds title of the survey form
- Adds the description of the survey form
 #### Description of Task to be completed?
- Sets up SASS in the project 
 #### How should this be manually tested?
- Clone the repo
- Navigate to the branch `ch-180944821-set-up-sass`
- Run `npm install`
- Run `compile:sass`
- If you make any changes to the sass files, you should see the changes to `#style.css`
 #### Any background context you want to provide?
- None
 #### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/180944821
- https://www.pivotaltracker.com/story/show/180876855
- https://www.pivotaltracker.com/story/show/180889100
 #### Screenshots (if appropriate)
- N/A
 #### Questions:
- None
#### Comments: 
- The added title and description are not styled, this will be done in another story.

### CODE REVIEW

**Comment by e-liyai** [comment]
@ginabeki it's not good practice to commit empty files. If you don't have content for the file at the moment, you can always omit it until you have content ready for it.

**Comment by ginabeki** [comment]
This is helpful. Thank you.

**Comment by e-liyai** [comment]
Also on the top-right menu, there's a reviewers button. Click the gear icon it gives a list of users that you want to review your work, otherwise this looks good to me.

**Review by e-liyai — APPROVED** [review]

**Comment by ginabeki** [comment]
> Also on the top-right menu, there's a reviewers button. Click the gear icon it gives a list of users that you want to review your work, otherwise this looks good to me.

@e-liyai Done

**Comment by ginabeki** [comment]
@e-liyai I've just followed your steps to make the adjustments, didn't notice you'd already fixed it. Kindly ignore the second request if that makes sense.

**Comment by ginabeki** [comment]
> @ginabeki it's not good practice to commit empty files. If you don't have content for the file at the moment, you can always omit it until you have content ready for it.

This is done. Added comments to the files describing their purpose.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (puritybirir) [inline .gitignore:1]: Add space after this line. EOL

More on this here => https://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/#:~:text=will%20be%20LF%20.-,core.,to%20denote%20end%20of%20line.

File: sass/_animation.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+// submit button animation elements will appear here
```
Comment (puritybirir) [inline sass/_animation.scss:1]: Add space as well here at the end of the line.

More on this here => https://adaptivepatchwork.com/2012/03/01/mind-the-end-of-your-line/#:~:text=will%20be%20LF%20.-,core.,to%20denote%20end%20of%20line.

**Review by puritybirir — CHANGES_REQUESTED** [review]
Apart from the End of line issues which are easily fixed, this Looks Good to me

File: .gitignore (hunk 1 above)
Comment (ginabeki) [reply .gitignore]: Fixed.

File: sass/_animation.scss (hunk 2 above)
Comment (ginabeki) [reply sass/_animation.scss:1]: fixed

**Review by puritybirir — APPROVED** [review]
LGTM!

**Pull request merged**